### ISSUE TITLE
Dress Rehearsal

### ISSUE THREAD

**Issue by iKAN2025** [iKAN2025/frontend#9]
USAGE OF:

 Frontend
 Backend
 API
 Agile
HOOK:
Points: 3.6-4.0
Reason:

KNOWLEDGE:
Points: 3.6-4.0
Reason:

VALUE:
Points: 0.6-1.0
Reason:

WOW FACTOR:
Reason

**Comment by jm1021** [comment]
USAGE OF:

- [X] Frontend
- [X]  Backend
- [x]  API
- [ ]  Agile

HOOK:
Points: 3.6-4.0
Reason:
Instrument tracker for music.  Hook was UI

KNOWLEDGE:
Points: 3.6-4.0
Reason:
Shared verification of data in Inspect.  But, not clear on what is happening.

VALUE:
Points: 0.6-1.0
Reason:
Has purpose but demo is still hard to follow.  Need to consider loading data before so it is less shaky.

WOW FACTOR:
Reason:
Graphs of tracking.

9.1/10

**Comment by Chrissiez** [comment]
USAGE OF:

- [x]  Frontend
- [x] Backend
- [x] API
- [x] Agile
HOOK:
Points: 3.8
Reason: good presentation, clear

KNOWLEDGE:
Points: 4.0
Reason: able to explain everything

VALUE:
Points: 0.8
Reason: good at tracking music

WOW FACTOR: alright
Reason: could have been more visually appealing

**Comment by srivaidyas** [comment]
USAGE OF:

- [X] Frontend
- [X] Backend
- [x] API
- [X] Agile

HOOK: Keeping track of your practice time
Points: 3.8
Reason: Doesn't pull you like it is supposed to. Good Idea.

KNOWLEDGE:
Points: 3.9
Reason: Demonstrated excellence in knowledge of backend and forntend. Included Parallelx.

VALUE:
Points: 0.95
Reason: Unique Idea helps put everything digital instead on paper

WOW FACTOR: 
Points: 0.92
Reason: Not one of a kind. Could have been more visually appealing.

**Comment by DanteA-tech** [comment]
USAGE OF:

Frontend
Backend
API
Agile

HOOK:
Points: 3.6
Reason: There wasn't really much of a hook, and wasn't too engaging.

KNOWLEDGE:
Points: 3.8
Reason:

VALUE:
Points: .9
Reason:
Seems very useful for beginners who are learning how to play and instrument, but I think the more advanced players wouldn't need it.
WOW FACTOR: The graphs
Reason: 1
Was very useful and shows the progress you are making. 
9.3/10

**Comment by DanteA-tech** [comment]
8.3/9

**Comment by AdiBricks** [comment]
Criteria/Key Features:

 Frontend - done
 Backend 0 - done
 API - done
 Agile - done

HOOK:
Points: 3.8
Reason: Interesting, definitely a useful design! 

KNOWLEDGE: 
Points: 4.0
Reason: Able to explain everything as asked, did a great job. 

VALUE:
Points: 1.0
Reason: definitely looks like something useful

Overall: 8.8/9

**Comment by Nupurbb** [comment]
Frontend
 Backend
 API
 Agile
HOOK:
Points: 3.7
Reason: good presentation
KNOWLEDGE:
Points: 3.9
Reason: able to explain everything

VALUE:
Points: 0.7
Reason: good at tracking music

WOW FACTOR: good

**Comment by vidhaganji** [comment]
Used frontend, backend, and an API

HOOK: 3.7
Music-relatred, kept me interested

KNOWLEDGE: 3.9

Had clear understand of topic


VALUE: 0.8

Music is source of recreation, can be important to society

WOW FACTOR: 0.7

Music-related topic

9.2/10

**Comment by vibhayg** [comment]
Frontend
Backend
API

HOOK: 3.9

Unique topic

KNOWLEDGE: 3.9
Explained very well to audience


VALUE: 0.8

Music is interesting to many people so is important for people

WOW FACTOR: 0.7

The graphs/charts

9.3/10

**Comment by parkib** [comment]
Frontend
Backend
API

HOOK: 3.9

Unique topic

KNOWLEDGE: 3.9
The knowledge was very well explained to the users- good job!!

VALUE: 0.8
Music is important to many artists and it changes their everyday lives.

WOW FACTOR: 0.7
The charts and graphs that were included were very cool.

9.3/10
